small updates (#856)
telerik/maui-docs: controls/scheduler/built-in-dialogs.md

Check below the available dialogs for managing appointments.

## Create and Edit Appointment

> The dialog which allows the end user create a new appointment or edit an existing one is the `EditAppointmentDialog`.

Reviewer: ```suggestion
`EditAppointmentDialog` is the dialog that allows the end user to create a new appointment or edit an existing one.
```

* On desktop, the `EditAppointmentDialog` appears when the end user double-clicks an empty slot or an appointment.

> * On mobile, the behavior of the `EditAppointmentDialog` depends on the selected calendar slot. If the end user taps an empty slot, the Edit Appointment dialog appears to let them create a new appointment. If the end user taps an existing appointment and then taps Edit in the Preview Appointment dialog, the Edit Appointment dialog appears to let them modify the appointment.

Reviewer: ```suggestion
* On mobile, the behavior of the `EditAppointmentDialog` depends on the selected calendar slot:
   * If the end user taps an empty slot, the Edit Appointment dialog appears to let them create a new appointment. 
   * If the end user taps an existing appointment and then taps Edit in the Preview Appointment dialog, the Edit Appointment dialog appears to let them modify the appointment.
```

* `DeleteRecurrenceChoiceDialogStyle`&mdash;Defines the `Style` that will be applied on the Delete Recurrence Choice dialog.
* `PreviewAppointmentDialogStyle`&mdash;(Android- and iOS-only)Defines the `Style` for the Preview Appointment dialog. 

> **Example with styling the edit appointment dialog**

Reviewer: Don't use bold as a subheading or a caption. Use as complete sentence instead.
```suggestion
The following example demonstrates how to style the Edit Appointment dialog.
```

**1.** Define the Scheduler with the `EditAppointmentDialogStyle` in XAML:

```XAML
 <telerik:RadScheduler x:Name="scheduler"
                       EditAppointmentDialogStyle="{StaticResource CustomSchedulerEditAppointmentDialog_Style}"/>
```

**2.** Define the style in the resources of the page:

```XAML
<Style x:Key="CustomSchedulerEditAppointmentDialog_Style" TargetType="telerik:SchedulerDialog">
    <Setter Property="ControlTemplate" Value="{StaticResource CustomSchedulerEditAppointmentDialog_ControlTemplate}" />
    <Setter Property="HeightRequest" Value="{OnPlatform Default=450, WinUI=530}" />
</Style>
```

> Review the `ControlTemplate` definition in the <a href="https://github.com/telerik/maui-samples/tree/main/Samples/ControlsSamples/Examples/SchedulerControl/CustomizationExample/Resources" target="_blank">Resources folder of the Controls Samples application > Scheduler/Customization example</a>.

Reviewer: You provide a direct link, so you don't need to describe the location word by word as the sentence gets too confusing.
```suggestion
Review the `ControlTemplate` definition in the <a href="https://github.com/telerik/maui-samples/tree/main/Samples/ControlsSamples/Examples/SchedulerControl/CustomizationExample/Resources" target="_blank">`Resources` folder of the Controls Samples application.
```
